core/corehttp/gateway_handler.go

```diff
@@ -133,6 +133,7 @@ func (i *gatewayHandler) getOrHeadHandler(w http.ResponseWriter, r *http.Request
 	}
 
 	w.Header().Set("X-IPFS-Path", urlPath)
+	w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
Is there any use-case for treating browsers differently than non-browser clients?